service/src/main/java/bio/terra/workspace/service/resource/controlled/flight/clone/dataset/CloneControlledGcpBigQueryDatasetResourceFlight.java

```diff
@@ -55,41 +56,61 @@ public CloneControlledGcpBigQueryDatasetResourceFlight(
       addStep(
           new SetNoOpBucketCloneResponseStep(
               sourceResource.castByEnum(WsmResourceType.CONTROLLED_GCP_BIG_QUERY_DATASET)));
-    } else if (CloningInstructions.COPY_RESOURCE == resolvedCloningInstructions
-        || CloningInstructions.COPY_DEFINITION == resolvedCloningInstructions) {
-      // Flight Plan
-      // 1. Validate user has read access to the source object
-      // 2. Gather controlled resource metadata for source object
-      // 3. Gather creation parameters from existing object
-      // 4. Launch sub-flight to create appropriate resource
-      addStep(
-          new CheckControlledResourceAuthStep(
-              sourceResource, flightBeanBag.getControlledResourceMetadataManager(), userRequest),
-          RetryRules.shortExponential());
-      if (mergePolicies) {
-        addStep(
-            new ClonePolicyAttributesStep(
-                sourceResource.getWorkspaceId(),
-                destinationWorkspaceId,
-                userRequest,
-                flightBeanBag.getTpsApiDispatch()));
-      }
+      return;
+    }
+
+    // Flight Plan
+    // 1. Validate user has read access to the source object
+    // 2. Gather controlled resource metadata for source object
+    // 3. Gather creation parameters from existing object
```

Comment:
i think here "gather cloud attributes" is easier for me to understand the difference between step 2 and 3.